codingStandards/CodingStandard-Java.md

```diff
@@ -385,23 +385,26 @@ The package statement location is enforced by the Java language. A Java package
 
 The rule of thumb is to package the classes that are related. For example in Java, the classes related to file writing is grouped in the package `java.io` and the classes which handle lists, maps etc are grouped in `java.util` package.
 
-**2. The import statements must follow the package statement. import statements should be sorted with the most fundamental packages first, and grouped with associated packages together and one blank line between groups.**
+**2. Import statements must follow the package statement.**
 
-```java
-import java.io.IOException;
-import java.net.URL;
-import java.rmi.RmiServer;
-import java.rmi.server.Server;
-import javax.swing.JPanel;
-import javax.swing.event.ActionEvent;
-import org.linux.apache.server.SoapServer;
-```
+The location of import statements is enforced by the Java language.
```

Comment:
Do you mean we don't even need this rule in the coding standard (because Java enforces it anyway)?